runner_option.go

```diff
@@ -327,3 +335,21 @@ func LocalForward(l string) sshRunnerOption {
 		return nil
 	}
 }
+
+func (t *traceConfig) UnmarshalYAML(b []byte) error {
+	if enable, err := strconv.ParseBool(strings.TrimSpace(string(b))); err == nil {
+		t.Enable = &enable
+		return nil
+	}
+
+	type TC traceConfig
```

Comment:
Does it have to be an uppercase variable?

I thought I could make `traceConfig` into `TraceConfig` if needed.

Reply:
I've tried to refer to this one.
https://github.com/k1LoW/octocov/blob/main/report/json.go